src/theory/sets/inference_manager.h

```diff
@@ -109,6 +109,9 @@ class InferenceManager
   /** Have we sent lem as a lemma in the current user context? */
   bool hasLemmaCached(Node lem) const;
 
+  /** Set conf is a conflict node to be sent on the output channel.  */
```

Comment:
It sounds a bit like two sentences got mixed up here. How about:

```suggestion
  /** 
   * Send conflict.
   * @param conf The conflict node to be sent on the output channel
   */
```